src/components/FormFieldWerkingsgebiedKoppeling/FormFieldWerkingsgebiedKoppeling.js

```diff
@@ -16,21 +16,17 @@ import PopupContainer from './../PopupContainer'
 import FormFieldTitelEnBeschrijving from '../FormFieldTitelEnBeschrijving/FormFieldTitelEnBeschrijving'
 
 /**
- * Component that renders the FormFieldWerkingsgebiedKoppeling component, that uses the imported components;
- * FormFieldTitelEnBeschrijving, to display the Title and description of the FormFieldWerkingsgebiedKoppeling component,
- * CardSelectedWerkingsgebied component, to disconnect the selected werkingsgebied and the WerkingsgebiedPopup component to select a werkingsgebied that the user wants connected.
- *
- * The imported transition component is used for the animation of part of the FormFieldWerkingsgebiedKoppeling component.
+ * Component that renders the FormFieldWerkingsgebiedKoppeling component that displays a title and description with a button in which the user can connect a Werkingsgebied.
  *
  * @component
  *
  * @param {function} setWerkingsgebiedInParentState - Parameter that is used to set the state value of the werkingsgebiedInparentState variable.
- * @param {function} werkingsgebiedInParentState - Parameter that is passed down from the parent and contains the UUID of the werkingsgebied object.
- * @param {string} dataObjectProperty - Parameter that is passed down from the parent and contains the werkingsgebied value of the werkingsgebied object.
- * @param {string} titleSingular - Parameter that is passed down from the parent and contains the title value of the werkingsgebied object that is used in the id of a div within this component and its value is passed down to the FormFieldTitelEnBeschrijving component.
- * @param {string} fieldLabel - Parameter that is passed down from the parent and contains the field label value of the werkingsgebied object, which value is passed down to the FormFieldTitelEnBeschrijving component.
- * @param {string} pValue - Parameter that is passed down from the parent and contains the paragraf value of the werkingsgebied object, which value is passed down to the FormFieldTitelEnBeschrijving component.
- * @param {boolean} disabled - Parameter that is passed down from the parent, which value is passed down to the FormFieldTitelEnBeschrijving component and used in this component in a conditional operator of a div to set a certain style value.
+ * @param {function} werkingsgebiedInParentState - Parameter that contains the UUID of the werkingsgebied object.
+ * @param {string} dataObjectProperty - Parameter containing the werkingsgebied data property.
```

Comment:
Parameter containing the werkingsgebied data property.
-> 
This field contains the property name for a geographical area, as this is not always the same on the crudObject.

Reply:
Thanks for the description change for the parameter. I've changed the description accordingly.